ci/py3.8.yml

```diff
@@ -28,7 +28,10 @@ dependencies:
   - scipy
   - setuptools
   - toolz
-  - xarray>=0.16.2
+#  - xarray>=0.16.2
   - zarr>=2.6.0
   - pip:
-    - git+https://github.com/rabernat/rechunker.git@refactor-executors
+    - git+https://github.com/rabernat/xarray.git@zarr-chunk-fixes
+    - git+https://github.com/pangeo-data/rechunker.git@master
+    - git+https://github.com/intake/filesystem_spec.git@master
+    - fsspec[http]
```

Comment:
@martindurant is this the right way to do this? The tests need `fsspec[http]`.

Reply:
yes, or just add the extras (requests, aiohttp) to the conda section

Reply:
Does it make sense that adding this line would trigger a 30+ minute "setting up miniconda" step?